web3-functions/redstone/index.ts

```diff
@@ -100,34 +110,40 @@ Web3Function.onRun(async (context: Web3FunctionContext) => {
       dataFeedIdsBytes32
     );
```

Comment:
Do we need to use `try`/`catch` block here?
Same question applies for `wrappedOracle.populateTransaction.updateDataFeedsValuesPartial` function call down below.

Reply:
I am not sure if it could catch something as the source function `_securelyExtractOracleValuesAndTimestampFromTxMsg` reverts without throwing if any of the required conditions do not match. Same for below.

I also do not see any additional value in using try catch at L115 below, other than emitting a different error message as the underlying function would also throw in case of error.